packages/p2p-v2/src/pages/orders/components/ChatHeader/ChatHeader.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { OnlineStatusLabel, UserAvatar } from '@/components';
+import { Text, useDevice } from '@deriv-com/ui';
+import './ChatHeader.scss';
+
+type TChatHeaderProps = {
+    isOnline: 0 | 1;
+    lastOnlineTime: number;
+    nickname: string;
+};
+const ChatHeader = ({ isOnline = 1, lastOnlineTime = 123143, nickname = 'asdf' }: TChatHeaderProps) => {
```

Comment:
why do we have `asdf` as the default value here?

Reply:
ohh.. i forgot to remove the dummy values.. will udpate.